woocommerce/woocommerce-android
# issue/3532-variations-crash-fix

Fixes #3532 by adding logic to reuse existing variation object when an item in the list is clicked. Again I wasn't really able to reproduce this one but since the crash is happening when a variation is clicked and we seem to fetch the variation from the list again a second time even though the variation object is available for us, I just updated that logic in the app.

### To test
- Click on a variable product from the app.
- Click on the Variations section.
- Notice the list os loaded successfully and you are able to click on aa variation to view the detail screen, without any issues.

Update release notes:
